From dfaa23858280861b9a6d4dc869b98293168ff467 Mon Sep 17 00:00:00 2001
From: Nils Freydank <nils.freydank@posteo.de>
Date: Sat, 23 Mar 2024 19:00:00 +0100
Subject: [PATCH] settings: disable checking for updates by default

Do not query the servers for updates by default on every
startup. The user can still enable this via GUI if desired.

This patch is nearly 1:1 the patch by Henning Schild from 2019-05-09.

Signed-off-by: Nils Freydank <holgersson@posteo.de>
---
 src/libs/core/settings.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/libs/core/settings.cpp b/src/libs/core/settings.cpp
index 8bcc6e6..bd81153 100644
--- a/src/libs/core/settings.cpp
+++ b/src/libs/core/settings.cpp
@@ -95,7 +95,8 @@ void Settings::load()
 
     // TODO: Put everything in groups
     startMinimized = settings->value(QStringLiteral("start_minimized"), false).toBool();
-    checkForUpdate = settings->value(QStringLiteral("check_for_update"), true).toBool();
+    // Disabled by default on Gentoo/Linux via custom patch.
+    checkForUpdate = settings->value(QStringLiteral("check_for_update"), false).toBool();
 
     showSystrayIcon = settings->value(QStringLiteral("show_systray_icon"), true).toBool();
     minimizeToSystray = settings->value(QStringLiteral("minimize_to_systray"), false).toBool();
-- 
2.44.0

